tools/launch_instance.rb

```diff
@@ -680,6 +680,12 @@ def launch_template(config:, launched_instances_name_prefix:)
       hosts = []
       terminate_spec = {}
       file_details = {}
+
+      # Export configuration's block of environment variables to running environment context
+      conf[:services, :"install-envvars"].each do |key, val|
```

Comment:
We need to think whether it should be under `services` or another place. e.g. under `global`

Reply:
Good point! Originally we agreed to do that by this way, so I'm following our agreement. 
But it make sense to extract it from *services* completely. Let touch this topic on mtg...

Reply:
As agreed, we moved the introduced section under `global`.

Brief overview of usage in `config.yaml`:
```
services:
  cloud1:
    ..
    ..
  cloudX:
    ..
    ..
global:
  install-envvars:
    key1: value1
    key2: value2
    ..
    keyZ: valueZ
```

Variables defined by above approach are exported and available in the running context for Ruby templates and Shell scripts.

@apodhrad Andrej, can you review and provide feedback if such an implementation fulfill you requirements please?